src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+  </head>
+  <body>
+    <section class="to-do-list">
```

Comment:
- [ ] [OPTIONAL] Good work on the app layout, however I suggest centring the todo lidt to be more semantic and easy to work with

Reply:
Thank you very much! Done!